environment.yml

```diff
@@ -14,3 +14,6 @@ dependencies:
   - sphinx-automodapi
   - sphinx_rtd_theme
   - vitables
+  - pip:
+    - Cython
```

Comment:
gammapy 0.18 has been released, and from the next version of pyirf we won't require cython anymore